_posts/2016-10-10-Help!-I'm-becoming-Post-Junior.md

```diff
@@ -0,0 +1,162 @@
+
+Key point:
+
+> Performance evaluation at Artsy is composed of the what, _i.e. what you achieve that contributes to your team’s goals and ultimately Artsy goals_, and the how, _i.e. how you act and how those actions contribute to Artsy’s culture and values_.
+
+We measure your career stage by your **impact** at Artsy, this is initially on the products you’re working on, but eventually moves out to the culture and the business:
+
+- **Engineer 1** - Can ship a well defined product feature.
+- **Engineer 2** - Can independently own a product features, and can handle the communication with others around it.
+- **Engineer 3** - Can handle a suite of features, and broadly contribute within a domain. Can improve company culture.
+- **Engineer 4** - Can improve and be a multiplier on other people’s work, can anticipate larger trends and affect culture to avoid or steer in that direction.
+- **Engineer 5** - Defines technical culture, works on impacting all parts of our businesses and creates new opportunities for the company.
+
+What we’re talking about here is the transition from Engineer 1, to Engineer 2. Here’s our full unabridged description of an Engineer 2.
+
+>  ▪	Consistently writes and delivers correct and clean quality code with guidance.
+
+> ▪	Self-sufficient and makes steady progress on tasks.
+
+> ▪	Knows when to ask for help and how to get unblocked.
+
+> ▪	Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
+
+> ▪	Owns a small-to-medium feature from technical design through completion.
+
+> ▪	Provides help and support outside of area under their responsibility.
+
+What can we gleam from this, that we start thinking of ways to improve ourselves as individual contributors?
+
+## Being Aware of How The Sausage Is Made
+
+It’s very unlikely that you are making all of the calls in a product, what is hopefully happening is that your team figures out a plan to ship something in a reasonable timeframe and as a team, you assign each other smaller talks that make that work. 
+
+It’s very likely that as a junior, you will be given the most well-defined small tasks. In OSS we call these the “[easy first steps][2].” tasks that can be done atomically, without requiring more interaction with designers or members of the product team. This is a feature, that eventually turns into a bug. 
+
+When it’s time to divvy up responsibilities, you should consider speaking up about taking tasks that are blocking people, but require further investigation outside of your dev team.
+
+This could be checking up occasionally with an API team, or leasing with different parts of the business to get confirmation on a specific detail. These kind of skills are not the sort of things you get in a computer science degree, and those without them are in a great place to other skills they’ve learned.
+
+## Shipping 
```

Comment:
Again, I would maybe call this "Awareness of Contextual History" / "Learn From Your Project's History" or something like that.